examples/FlinkDeployment.yaml

```diff
@@ -3,7 +3,7 @@ kind: FlinkDeployment
 metadata:
   name: flink-deployment-example
 spec:
-  image:  flink-sql-runner:latest
+  image: localhost/flink-sql-runner:latest
```

Comment:
I think this should point to quay registry and org where the real image from CI pipeline will be placed